libpod/common/common.go

```diff
@@ -61,22 +57,38 @@ func IsValidBool(str string) bool {
 	return IsTrue(str) || IsFalse(str)
 }
 
+// Helper function to determine the username/password passed
+// in the creds string.  It could be either or both.
+func parseCreds(creds string) (string, string) {
+	if creds == "" {
+		return "", ""
+	}
+	up := strings.SplitN(creds, ":", 2)
+	if len(up) == 1 {
+		return up[0], ""
+	}
+	return up[0], up[1]
+}
+
 // ParseRegistryCreds takes a credentials string in the form USERNAME:PASSWORD
 // and returns a DockerAuthConfig
 func ParseRegistryCreds(creds string) (*types.DockerAuthConfig, error) {
```

Comment:
Same here - this seems like a good opportunity to move the code to someplace more actively maintained